generatebundlefile/main.go

```diff
@@ -36,6 +37,88 @@ func main() {
 		return
 	}
 
+	if o.promote != "" {
+		fmt.Printf("Promoting %s from private ECR to Public ECR\n", o.promote)
+		// Get AWS Clients
+		ecrPublicClient, err := NewECRPublicClient(true)
+		if err != nil {
+			BundleLog.Error(err, "Unable to create SDK connection to ECR Public")
+		}
+		ecrClient, err := NewECRClient(true)
+		if err != nil {
+			BundleLog.Error(err, "Unable to create SDK connection to ECR")
+		}
+		stsClient, err := NewStsClient(true)
+		if err != nil {
+			BundleLog.Error(err, "Unable to create SDK connection to STS")
+		}
+		name, version, err := ecrClient.getNameAndVersion(o.promote, stsClient.AccountID)
+		fmt.Printf("Promoting chart and image version %s:%s\n", name, version)
+		semVer := strings.Replace(version, "_", "+", 1) // TODO use the Semvar library instead of this hack.
```

Comment:
Actually, helm should do this for you, I believe.

Reply:
Do you have reference for how to do that?

Reply:
No reference, just remove the `string.Replace`, helm will do that... Assuming you're using helm to push/pull the packages at least. If you're doing that via skopeo or oras or something else, then yeah, you'll need this.